gradle/libs.versions.toml

```diff
@@ -29,6 +34,11 @@ kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-reflect", version.ref =
 kotlin-stdlib-jdk8 = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk8", version.ref = "kotlin" }
 system-lambda = { module = "com.github.stefanbirkner:system-lambda", version.ref = "system-lambda" }
 
+# TestContainers
+
+testcontainers-bom = { module = "org.testcontainers:testcontainers-bom", version.ref = "testcontainers" }
```

Comment:
You don't need this BOM. See https://github.com/micronaut-projects/micronaut-object-storage/blob/master/gradle/libs.versions.toml as an example

Reply:
Fixed